chanbackup/pubsub.go

```diff
@@ -180,6 +196,44 @@ func (s *SubSwapper) Stop() error {
 	return nil
 }
 
+// ManualUpdate inserts/updates channel states into the swapper. The updates
+// are processed in another goroutine. If wait=true, the method waits for the
+// updates to be fully processed and the file updated on-disk before returning.
+func (s *SubSwapper) ManualUpdate(singles []Single, wait bool) error {
```

Comment:
I think `wait` is always true in how we currently use this. Should we just remove that param then to simplify things a bit?

Reply:
Done.